### DISCUSSION TITLE
为何再自己训练完文字识别模型后，测试结果返回 ppocr INFO:        result:        0.0

### DISCUSSION THREAD

**Discussion by SolarKnight1** [PaddlePaddle/PaddleOCR#12668] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
 运行指令如下python  .\tools\infer_rec.py -c C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\configs\rec\PP-OCRv4\ch_PP-OCRv4_rec.yml  -o Global.pretrained_model=C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\output\rec_ppocr_v4\best_accuracy.pdparams Global.infer_img="C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\1.png"

其中model是ocrv4的量化版经过了5000条cdla数据集训练出的模型，图片是正常pdf图片，为什么会啥结果都不返回啊，直接一个0

- 系统环境/System Environment：
- 版本号/Version：Paddle：2.5.1.post120  PaddleOCR：2.7.0.2  问题相关组件/Related components：系统windows  显卡a10  cuda12.2
- 运行指令/Command Code：python  .\tools\infer_rec.py -c C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\configs\rec\PP-OCRv4\ch_PP-OCRv4_rec.yml  -o Global.pretrained_model=C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\output\rec_ppocr_v4\best_accuracy.pdparams Global.infer_img="C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\1.png"
- 完整报错/Complete Error Message：
ppocr INFO: infer_img: C:\Users\Administrator\PycharmProjects\pythonProject\PaddleOCR\1.png
ppocr INFO:        result:        0.0
 ppocr INFO: success!

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by SWHL** [comment]
感觉是哪里没有指定导致的，从反馈来看，猜不出来是哪里问题
